packages/components/src/components/radio-button/radio-button.tsx

```diff
@@ -44,6 +44,7 @@ export class RadioButton {
   render() {
     return (
       <div
+        aria-label='a radio button' aria-value={this.value} aria-disabled={this.disabled}
```

Comment:
here as well. would it be better practice to have the aria label as a prop and have the user set it? we need to align more on the accessibility topic